projects/addon-editor/components/edit-link/test/edit-link-parse.spec.ts

```diff
@@ -0,0 +1,160 @@
+        });
+    });
+
+    it(`mailto`, () => {
+        expect(tuiEditLinkParseUrl(`mailto:user1@example.com`)).toEqual({
+            prefix: `mailto:`,
+            path: `user1@example.com`,
+        });
+
+        expect(
+            tuiEditLinkParseUrl(
+                `mailto:user1@example.com%2C%20user2@example.com?to=user3@example.com%2C%20user4@example.com`,
+            ),
+        ).toEqual({
+            prefix: `mailto:`,
+            path: `user1@example.com%2C%20user2@example.com?to=user3@example.com%2C%20user4@example.com`,
+        });
+    });
+
+    it(`custom protocol schema`, () => {
+        expect(tuiEditLinkParseUrl(`web+burger:Burger handler`)).toEqual({
+            prefix: `web+burger:`,
+            path: `Burger handler`,
+        });
+    });
+
+    it(`without protocol`, () => {
+        expect(tuiEditLinkParseUrl(`hello.com`)).toEqual({
+            prefix: ``,
+            path: `hello.com`,
+        });
+
+        expect(tuiEditLinkParseUrl(``)).toEqual({prefix: ``, path: ``});
+    });
+
+    it(`anchor`, () => {
+        expect(tuiEditLinkParseUrl(`#path-to-anchor`)).toEqual({
+            prefix: `#`,
+            path: `path-to-anchor`,
+        });
+    });
+
+    it(`custom value`, () => {
+        expect(tuiEditLinkParseUrl(`://hello.com`)).toEqual({
+            prefix: `://`,
+            path: `hello.com`,
+        });
+
+        expect(tuiEditLinkParseUrl(`//hello.com`)).toEqual({
+            prefix: ``,
+            path: `//hello.com`,
+        });
+
+        expect(tuiEditLinkParseUrl(`/hello.com/a/b/c`)).toEqual({
+            prefix: ``,
+            path: `/hello.com/a/b/c`,
+        });
+    });
+
+    it(`duplicate protocols`, () => {
+        expect(tuiEditLinkParseUrl(`http://tel:+79001111111`)).toEqual({
+            prefix: `tel:`,
+            path: `+79001111111`,
+        });
+
+        expect(tuiEditLinkParseUrl(`https://ssh`)).toEqual({
+            prefix: `https://`,
+            path: `ssh`,
+        });
+
+        expect(tuiEditLinkParseUrl(`https://ssh:`)).toEqual({
+            prefix: `https://`,
+            path: `ssh:`,
+        });
+
+        expect(tuiEditLinkParseUrl(`https://ssh:/`)).toEqual({
+            prefix: `ssh:`,
+            path: `/`,
+        });
+
+        expect(tuiEditLinkParseUrl(`https://ssh://`)).toEqual({
+            prefix: `ssh://`,
+            path: ``,
+        });
+
+        expect(tuiEditLinkParseUrl(`https://ssh://abc`)).toEqual({
+            prefix: `ssh://`,
+            path: `abc`,
+        });
+
+        expect(tuiEditLinkParseUrl(`https://a://b://c://d://e`)).toEqual({
+            prefix: `d://`,
+            path: `e`,
+        });
+
+        expect(tuiEditLinkParseUrl(`a:b:c:d:e`)).toEqual({
+            prefix: `d:`,
+            path: `e`,
+        });
+    });
+
+    it(`URLs`, () => {
+        expect(tuiEditLinkParseUrl(`https://taiga-ui.dev`)).toEqual({
+            prefix: `https://`,
+            path: `taiga-ui.dev`,
```

Comment:
You have two identical tests in one file.
Is it ok ?

Reply:
fixed